redfish-core/lib/processor.hpp

```diff
@@ -902,117 +902,165 @@ inline void getProcessorData(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
 }
 
 inline void getCpuCoreDataByService(std::shared_ptr<bmcweb::AsyncResp> aResp,
-                                    const std::string& service,
+                                    const std::string& /* service */,
```

Comment:
Removed this patch (commit) since IPS folk addressed the bug in the upstream patch (I have refreshed the latest patch - https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/38570/37 patchset)